site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,271 @@
+</figure>
+
+Using these APIs, front-end JavaScript can detect and act upon prerendered pages appropriately.
+
+{% Aside 'warning' %}
+  At present the [`PerformanceNavigationTiming.type`](https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming/type) is not using the `prerender` value, that was for the older, original `<link rel="prerender"...>` navigation type. This is liable to change in future but for now, clients should use the `document.prerendering` API to identify prerendering pages and a non-zero `PerformanceNavigationTiming.activationStart` field to identify prerendered pages that have since been activated.
+{% endAside %}
+
+## Impact on analytics
+
+Analytics is used to measure website usage, for example using Google Analytics to measure page views, and events. Or by measuring performance metrics of pages using [Real User Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring).
+
+Pages should only be prerendered when there is a high probability the page will be loaded by the user. This is why the Chrome address bar prerendering options only happen when there is such a high probability (80% or higher).
+
+However, particularly when using the Speculation Rules API, prerendered pages may have an impact on analytics and site owners may wish to add extra code to only enable analytics for prerendered pages on activation.
+
+This could be achieved by using a `Promise` which waits for the `prerenderingchange` event if a document is prerendering, or resolves immediately if it is now:
+
+```js
+// Set up a promise for when the page is activated,
+// which is needed for prerendered pages.
+const whenActivated = new Promise((resolve) => {
+  if (document.prerendering) {
+    document.addEventListener('prerenderingchange', resolve);
+  } else {
+    resolve();
+  }
+});
+
+async function initAnalytics() {
+  await whenActivated;
+  gtag('config', 'UA-12345678-1');
+}
+
+initAnalytics();
+```
+
+This is the method we have implemented on this site and on [https://web.dev](https://web.dev).
+
+### Measuring performance
+
+For measuring performance metrics, analytics should consider whether it is better to measure these based upon the activation time rather than the page load time that browser APIs will report.
+
+For Core Web Vitals, measured by Chrome through the [Chrome User Experience Report](/docs/crux/), these are intended to measure the user experience. So these are measured based on activation time. This will often result in a 0 second LCP for example, showing this is great way of improving your Core Web Vitals.
+
+From version 3.0.4, the [`web-vitals` library](https://github.com/GoogleChrome/web-vitals) has been updated to handle prerendered navigations in the same way Chrome measure Core Web Vitals. This version also flags prerendered navigations for those metrics in the [`Metric.navigationType`](https://github.com/GoogleChrome/web-vitals#metric) attribute, which is discussed next.
```

Comment:
I'd recommend people use at least `3.1.0` since some additional prerendering issues where fixed in that version: https://github.com/GoogleChrome/web-vitals/blob/main/CHANGELOG.md#v310-2022-11-15